app/services/cordova.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+
+// from https://cordova.apache.org/docs/en/4.0.0/cordova_events_events.md.html
+const CORDOVA_EVENTS = Ember.A([
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).